bodhi/server/validators.py

```diff
@@ -191,15 +192,15 @@ def cache_nvrs(request, build):
 
 
 @postschema_validator
-def validate_nvrs(request, **kwargs):
+def validate_build_nvrs(request, **kwargs):
     """
     Ensure that the given builds reference valid Build objects.
 
     Args:
-        request (pyramid.util.Request): The current request.
+        request (pyramid.request.Request): The current request.
         kwargs (dict): The kwargs of the related service definition. Unused.
     """
-    for build in request.validated.get('builds', []):
+    for build in request.validated.get('builds') or []:  # cope with builds being None
```

Comment:
This would be more idiomatic:

```suggestion
    for build in request.validated.get('builds', []):
```

Reply:
But it wouldn't be the same, that's why I replaced it (I want it to not croak at the `'builds'` key having `None` as a value). I could pull it apart like this to make it stand out more:

```
builds = request.validated.get('builds')
# cope with builds being None
if not builds:
    builds = []
for build in builds:
    ...
```

Reply:
@bowlofeggs I didn't make it clear this was a question: should I change this as I outlined, or add a comment?